_posts/en/posts/2016-01-21-segwit-testnet.md

```diff
@@ -0,0 +1,67 @@
+
+This significant innovation leads to dramatically more efficient use of the Bitcoin blockchain, while simultaneously opening up exciting new possibilities and opportunities for the broader Bitcoin ecosystem, particularly around smart contract applications and dramatically faster transactions, while providing the groundwork for more advanced features and possibilities in the future.
+
+This development began with a research effort by Bitcoin Core developer, Dr. Pieter Wuille, initially focused on addressing transaction malleability, a longstanding and well-known concern and priority. However, in the process of this research, and in the narrowing toward a solution, additional properties of the solution were discovered that allow for increasing the block size while also simultaneously opening up some incredibly exciting secondary benefits.
+
+This effort was initiated by Dr. Pieter Wuille, but included contributions from many others, with particular thanks to Eric Lombrozo, Johnson Lau, Alex Morcos, Nicolas Dorier, Bryan Bishop, Gregory Maxwell, Peter Todd, Cory Fields, Sudhas Daftuar, and Luke-Jr.
+
+## Bitcoin Core Ecosystem
+
+There is broad excitement and anticipation as far as what providers and other exchange operators will create with the fundamental developments and innovations included in this release. So far, the [most popular wallets][segwit_adoption] and supporting libraries have stated they will support segwit including Ledger, Trezor, Electrum, and Bitgo. Additionally, work on numerous other libraries such as bitcoinj, bitcoinjs, pycoin and bitcore has already begun.
+
+A faucet is available for "segnet" coins [here](https://segwit.greenaddress.it/faucet/).
+
+Early previews of third party wallet support are available at:
+
+- [mSIGNA](https://github.com/ciphrex/mSIGNA/tree/segwit3) (wallet source-code)
+- [Green Address](https://segwit.greenaddress.it/) (web wallet)
+
+## How to get involved
+
+If you are a wallet user interested in following the status of your wallet’s implementation, please [follow their progress][segwit_adoption] which will be regularly updated. The testing period will proceed according to the Bitcoin Core roadmap. If you would also like to list your wallet or library support, you can do so [here](https://github.com/bitcoin-core/website/pull/30).
+
+If you are a wallet provider, and interested in instructions for how to upgrade to a segwit-compatible wallet, please read the [migration guide](/en/segwit_wallet_dev)
+
+## Testing
+
+Finally and most importantly, please help testing the Segwit Testnet!
+
+Once compiled from the segwit branch, add `-segnet` to the standard `bitcoind` and `bitcoin-cli` command line.
```

Comment:
It's not clear to which "segwit branch" you refer here. Might be considered to put a link into the "early preview" section.